fades/envbuilder.py

```diff
@@ -38,6 +38,14 @@
 logger = logging.getLogger(__name__)
 
 
+class FadesError(Exception):
+    """Provides a Fades exception."""
+
+    def __init__(self, reason):
```

Comment:
This is not really needed, as Exceptions already receive and store a `message`, so just define FadesError, with the docstring, and that's all.

That said, it's better to define this exception in a more general place... put it in `fades/__init__.py` and import it from there...